### PR TITLE
Add new Company - Remote Ok

### PR DESCRIPTION
**Pull request by Teddir**
Signed-off-by: Teddi_r <70432343+Teddir@users.noreply.github.com>

This Pull Request should adhere to the repository's [Code of Conduct](../../main/.github/CONTRIBUTING.md).

Title your pull request descriptively. `[edit X company]` or `[add new company - COMPANY]` 

Please read and check the boxes in the list below that apply!

If updating or adding a company, please include details here: _delete this and add details_

Check off this list as appropriate (add an `x` between the `[]`). It will be used by @dougaitken when reviewing your contribution.

- [] This PR contains housekeeping only (URL edits, copy changes etc) - **if this is checked, delete other lines that don't apply.**
- [] You are an employee of the company mentioned and confirm all included details are correct.
- [] You know your alphabet - company is listed in alphabetical order in the README.md file.
- [] The company directly hires employees. ** No bootcamps / freelance sites / etc**
- [] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such.
- [] A [company profile](/company-profiles/example.md) is included - This is __required__ for new additions.
- [] An entry has been added or edited within the [readme.md file](../../main/README.md).
- [] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [] __Region__ details the geographic regions in which this company employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

This is a modified version of the [Contributing Guidelines](../../main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: company-profiles/remoteok.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+## Company blurb
```
Comment (KieranRobson) [inline company-profiles/remoteok.md:1]: The name of the company should be above `Company blurb` i.e `# Remote Ok`

File: README.md [hunk 2]
```diff
@@ -531,6 +531,7 @@ Name | Website | Region
 [RenoFi](/company-profiles/renofi.md) | https://renofi.com/ |  Worldwide
 [Research Square](/company-profiles/research-square.md) | https://www.researchsquare.com/ | USA
 [Revolut](/company-profiles/revolut.md) | https://www.revolut.com/ | Worldwide
+[Remote Ok](/company-profiles/remoteok.md) | https://www.remoteok.com/ | Worldwide
```
Comment (KieranRobson) [inline README.md:534]: This should be before `Revolut`

**Review by KieranRobson — COMMENTED** [review]
Logs are failing due to:

File: README.md [hunk 3]
```diff
@@ -531,6 +531,7 @@ Name | Website | Region
 [RenoFi](/company-profiles/renofi.md) | https://renofi.com/ |  Worldwide
```
Comment (KieranRobson) [inline README.md:531]: Remote OK Should be before here

File: company-profiles/remoteok.md (hunk 1 above)
Comment (KieranRobson) [inline company-profiles/remoteok.md:3]: This .md file should be renamed to `remote-ok.md`

**Comment by KieranRobson** [comment]
`9:25:06 AM: README.md: Company is listed out of order: "Remote Ok" (should be before "RemoteBase")` 

Turns out the space makes it so that `Remote Ok` should be before `RemoteBase`

**Comment by KieranRobson** [comment]
See the above comment

**Comment by Teddir** [comment]
> See the above comment

File: company-profiles/remote-ok.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+None
+
+## Office locations
+
+None
+
+## How to apply
+
+You can find our open jobs and apply directly here: [https://remoteok.com/](https://edg.io/company/careers)
```
Comment (KieranRobson) [inline company-profiles/remote-ok.md:29]: The URL needs to be updated from the one you used as a template

**Comment by KieranRobson** [comment]
Although I cant officially approve, I believe that's all that needed changing!

**Comment by Teddir** [comment]
> As I am not a maintainer of this project, I cant officially approve, I believe that's all that needed changing!

Thank you very much for helping to fix it

**Comment by dougaitken** [comment]
@Teddir To be clear - is the company Remote OK hiring directly, or are you adding the site as it is a job board?

**Comment by Teddir** [comment]
> @Teddir To be clear - is the company Remote OK hiring directly, or are you adding the site as it is a job board?
> 
> This list is for companies hiring directly and not for listing job boards.

ouhh, this is job board.  sorry i think this repo to all remotes job. 
maybe i will add another one. that match the repo criteria.
thank you @dougaitken

**Comment by dougaitken** [comment]
I can probably make this more obvious during contribution. Thanks @Teddir

**Pull request closed without merging**